diff -ruN rt205/include/spacemit_ort_env_c_api.h rt206/include/spacemit_ort_env_c_api.h
--- rt205/include/spacemit_ort_env_c_api.h	2026-07-03 12:06:19.000000000 +0200
+++ rt206/include/spacemit_ort_env_c_api.h	2026-07-24 09:03:52.000000000 +0200
@@ -7,8 +7,8 @@

 // clang-format off
 #ifndef SpaceMITPROVIDER_VERSION
-#define SpaceMITPROVIDER_VERSION "2.0.5"
-#define SpaceMITPROVIDER_BUILD_DATE "2026-07-03"
+#define SpaceMITPROVIDER_VERSION "2.0.6"
+#define SpaceMITPROVIDER_BUILD_DATE "2026-07-24"
 #endif

 #ifdef __cplusplus
diff -ruN rt205/include/spacemit_ort_env.h rt206/include/spacemit_ort_env.h
--- rt205/include/spacemit_ort_env.h	2026-07-03 12:06:19.000000000 +0200
+++ rt206/include/spacemit_ort_env.h	2026-07-24 09:03:52.000000000 +0200
@@ -8,8 +8,8 @@
 #include "onnxruntime_cxx_api.h"
 // clang-format off
 #ifndef SpaceMITPROVIDER_VERSION
-#define SpaceMITPROVIDER_VERSION "2.0.5"
-#define SpaceMITPROVIDER_BUILD_DATE "2026-07-03"
+#define SpaceMITPROVIDER_VERSION "2.0.6"
+#define SpaceMITPROVIDER_BUILD_DATE "2026-07-24"
 #endif

 namespace Ort {
